restify/restify.d.ts

```diff
@@ -452,6 +452,7 @@ declare module "restify" {
     responseTimeFormatter ?: (durationInMilliseconds: number) => any;
     handleUpgrades ?: boolean;
     router ?: Router;
+    httpsServerOptions?: Object;
```

Comment:
Thanks for the input! Could you explain why `any` is preferred over `Object`?

The `httpsServerOptions` property should be undefined or null or only be given an Object as a value. If the property exists, Restify passes the the property into https.createServer [[1]](https://github.com/restify/node-restify/blob/5.x/lib/server.js#L291) which expects an Object [[2]](https://github.com/nodejs/node/blob/master/lib/https.js#L12).